app/src/main/java/com/cp/tasky/auth/login/presentation/LoginScreen.kt

```diff
@@ -159,6 +153,26 @@ private fun LoginScreen(
                         )
                     })
 
+                if (networkState is NetworkResult.Error) {
+                    Text(
+                        modifier = Modifier
+                            .padding(
+                                top = LARGE_16_DP,
+                                start = LARGE_16_DP,
+                                end = LARGE_16_DP
+                            )
+                            .align(Alignment.Start),
+                        text = "${(networkState as? NetworkResult.Error<LoginResponse>)?.exception}",
+                        color = Color.Red,
+                        maxLines = 2,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                    Log.d(
+                        "LoginScreen",
+                        "Exception: ${(networkState as? NetworkResult.Error<LoginResponse>)?.exception}"
+                    )
```

Comment:
This is a side-effect and the reason you see all those logs from the video. If you put a non-Compose function inside a composable function, it will be executed with every single recomposition (in this case with every single state update where networkState is an error). To properly log this, put this code in your ViewModel or inside an effect handler:
```
LaunchedEffect(true) {
    Log.d(...)
}
```

Reply:
But the LaunchedEffect{ } I won't be able to create circularProgressingBar composable in it. The only way to go is to create conditional composable inside the screen composable.